yarn.lock

```diff
@@ -1164,6 +1164,22 @@
   resolved "https://registry.yarnpkg.com/@types/vscode/-/vscode-1.45.0.tgz#8a07e740aa1ca7264da8cb17be87b733836d6c01"
   integrity sha512-b0Gyir7sPBCqiKLygAhn/AYVfzWD+SMPkWltBrIuPEyTOxSU1wVApWY/FcxYO2EWTRacoubTl4+gvZf86RkecA==
 
+"@xml-tools/ast@4.1.0":
+  version "4.1.0"
+  resolved "https://registry.yarnpkg.com/@xml-tools/ast/-/ast-4.1.0.tgz#2660783c9f9a600ed0c9e33bf8919542fb2be531"
+  integrity sha512-8nErsCzkk8PtRI3XP+FgSs3ZINtlR+Jr39zO73xc23VX8ASaNmdEtRgjWVXHC8aWGU1KAw+BWXmB2fVz19YCXQ==
+  dependencies:
+    "@xml-tools/common" "^0.0.6"
+    "@xml-tools/parser" "^1.0.4"
```

Comment:
this is actually not good, it is due to having dependency to none master version of the parser, so the mono repo works with two different versions of @xml-tools/parser (one from master and one from npmjs). This is fixed in: https://github.com/SAP/xml-tools/pull/182